c/heap-manipulation/dll_of_dll_false-unreach-call_false-valid-memcleanup.i

```diff
@@ -1,97 +1,13 @@
 extern void __VERIFIER_error() __attribute__ ((__noreturn__));
-typedef unsigned int size_t;
-typedef long int wchar_t;
+typedef long unsigned int size_t;
```

Comment:
I think you preprocessed these files with the wrong architecture, thus the change in the definition of `size_t` and the errors produced by `clang` on Travis.

Reply:
Thanks @PhilippWendler , I'm fighting with the checks, hope to have it ready today 😉